pkg/login/social/connectors/google_oauth.go

```diff
@@ -24,13 +24,16 @@ const (
 	legacyAPIURL            = "https://www.googleapis.com/oauth2/v1/userinfo"
 	googleIAMGroupsEndpoint = "https://content-cloudidentity.googleapis.com/v1/groups/-/memberships:searchDirectGroups"
 	googleIAMScope          = "https://www.googleapis.com/auth/cloud-identity.groups.readonly"
+	disableHDValidationKey  = "disable_hd_validation"
 )
 
 var _ social.SocialConnector = (*SocialGoogle)(nil)
 var _ ssosettings.Reloadable = (*SocialGoogle)(nil)
+var ExtraGoogleSettingKeys = []string{disableHDValidationKey}
 
 type SocialGoogle struct {
 	*SocialBase
+	disableHDValidation bool
```

Comment:
avoid negative options

rename to validateHD bool will make it more readable

Reply:
I have updated the variable name as requested.